static/result.html

```diff
@@ -1,216 +1,527 @@
+            font-weight: 200;
+        }
+
+        .italic {
+            font-style: italic;
+        }
+
+        .columns {
+            width: 100%;
+            column-count: 2;
+            column-gap: 1ch;
+        }
+
+        .noselect {
+            user-select: none;
+        }
+
+        .break-anywhere {
+            line-break: anywhere;
+        }
+
+        html,
+        body {
+            margin: 0;
+            background: var(--c-black);
+            color: var(--c-white);
+            min-height: 100vh;
+            font-family: var(--font);
+            font-weight: 400;
+            line-height: var(--line);
+            letter-spacing: 0.01em;
+        }
+
+        .separator-char {
+            display: none;
+        }
+
+        @media (min-width: 768px) {
+            body {
+                font-size: 2*0.7vw;
+            }
+
+            .separator-char {
+                display: contents;
+            }
+        }
+
+        @media (min-width: 1024px) {
+            body {
+                font-size: 1vw;
+                overflow: auto;
+
+                .separator-char {
+                    display: contents;
+                }
+            }
+
+            .large {
+                font-size: 4vw;
+                line-height: 1.11111111em;
+                font-weight: 200;
+            }
+        }
+
+        ::selection {
+            color: #000000;
+            background: var(--c-grey);
+        }
+
+        button {
+            display: inline;
+            font-family: inherit;
+            font-weight: inherit;
+            font-size: inherit;
+            letter-spacing: inherit;
+            padding: 0;
+            margin: 0;
+            background: transparent;
+            border: none;
+            outline: none;
+        }
+
+        .hover:hover {
+            cursor: pointer;
+        }
+
+        .pointer-events-none {
+            pointer-events: none;
+        }
+
+        body.invert canvas {
+            opacity: 0.05;
+            filter: none;
+            mix-blend-mode: normal;
+        }
+
+        .subtle-hover:hover {
+            background: transparent;
+            box-shadow: none;
+            cursor: pointer;
+            opacity: 0.8;
+            color: currentColor;
+        }
+
+        .title {
+            font-weight: lighter;
+        }
+
+        textarea {
+            min-height: 20rem;
+        }
+
+        #comment {
+            min-height: 5rem;
+        }
+
+        .vim-folded {
+            white-space: nowrap;
+            overflow-x: hidden
+        }
+
+        .vim-folded::before {
+            content: "+--\00A0";
+            white-space: nowrap;
+        }
+
+        .vim-folded::after {
+            display: inline;
+            content: "\00A0··························································································································································";
+            white-space: nowrap;
```

Comment:
Don't sure, but looks like you mix css and tailwind in case, when we can leave only tailwind.